tests/admin_filters/tests.py

```diff
@@ -140,12 +140,24 @@ class CustomUserAdmin(UserAdmin):
 
 
 class BookAdmin(ModelAdmin):
-    list_filter = ('year', 'author', 'contributors', 'is_best_seller', 'date_registered', 'no')
+    list_filter = ('year', 'author', 'contributors', 'is_best_seller', 'date_registered', 'no', 'availability')
     ordering = ('-id',)
 
 
 class BookAdmin2(ModelAdmin):
-    list_filter = ('year', 'author', 'contributors', 'is_best_seller2', 'date_registered', 'no')
+    list_filter = ('year', 'author', 'contributors', 'is_best_seller2', 'date_registered', 'no', 'availability')
+
+
+class BookAdminWithChoicesBooleanFilter(BookAdmin):
+    list_filter = (
+        'year',
+        'author',
+        'contributors',
+        'is_best_seller',
+        'date_registered',
+        'no',
+        ('availability', BooleanFieldListFilter),
+    )
```

Comment:
I would add `('availability', BooleanFieldListFilter)` to the `BookAdminWithTupleBooleanFilter` instead of creating a separate class.

Reply:
Removed the `BookAdminWithChoicesBooleanFilter` and added `('availability', BooleanFieldListFilter)`to the `BookAdminWithTupleBooleanFilter` .